src/components/Calendar/Weekdays.tsx

```diff
@@ -9,19 +9,30 @@ export const FORMAT = 'DD/MM/YYYY';
 export interface Props {
   week?: dayjs.Dayjs[];
   onSelect?: (selected: dayjs.Dayjs) => void;
-  selected?: dayjs.Dayjs | null;
+  selected: dayjs.Dayjs[];
   state?: any;
 }
 
 function Weekdays({ week = [], onSelect = () => {}, selected, state = {} }: Props) {
-  return (
+  const isSelected = (day: dayjs.Dayjs) => (selected.length > 1 &&
+      (selected[0].format(FORMAT) === day.format(FORMAT) ||
+        selected[1].format(FORMAT) === day.format(FORMAT))) ||
+      (selected!.length === 1 && selected[0].format(FORMAT) === day.format(FORMAT)) || false;
+  
+return (
     <div className="grid grid-cols-7 gap-x-6 mt-2">
       {week.map((day) => (
         <Day
           key={day.format(FORMAT)}
           day={day}
-          selected={day.format(FORMAT) === selected?.format(FORMAT)}
-          textColor={getTextColor(day, selected)}
+          selected={
+            isSelected(day)
+          }
+          inRange={
+            (selected.length > 1 && day > selected[0] && day < selected[1]) ||
+            false
+          }
```

Comment:
you can also look at the docs of dayjs if there is better/cleaner way to check if a date is in range

Reply:
@robdembitel yes there is but for that we need to add isBetween plugin of dayjs.  It will look something like:

` selected.length > 1 && day > selected[0] && day < selected[1]`

`selected.length > 1 && day.isBetween(selected[0], selected[1])`

should we do this if we have to install plugin?

Reply:
I think the plugin is shipped with the dayjs package anyway, you just have to require/import it seperately. So you can use it, since we already depend on dayjs.